### PR TITLE
feat: Add headless chromium to editor Dockerfile

### PR DESCRIPTION
**Pull request by vhf**
For use in https://github.com/upfrontIO/livingdocs-editor/pull/1711  , once merged.

### CODE REVIEW

**Comment by gabrielhase** [comment]
@marcbachmann would be great if you could find time to review this. It's the blocker for merging the server-side rendering PR from Lukas and unless we have that we can not make published npm packages out of our SDK projects.

**Comment by DaRaFF** [comment]
This is awesome! I had the plan to work on that today, but now you're my hero :-)

File: editor/Dockerfile [hunk 1]
```diff
@@ -2,15 +2,34 @@
...
 FROM node:8-alpine
 
-# Workaround for `npm publish` issue where test directories
-# are excluded from the published module in npm <v5.5
-# We use yarn here as a workaround for some docker file system issue
-RUN yarn global add npm@^5.5 && rm -Rf /usr/local/lib/node_modules/npm
+# Update apk repositories
+RUN echo "http://dl-2.alpinelinux.org/alpine/edge/main" > /etc/apk/repositories
+RUN echo "http://dl-2.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories
+RUN echo "http://dl-2.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
```
Comment (marcbachmann) [inline editor/Dockerfile]: I'd prefer if you could use
`RUN echo "@testing http://dl-2.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories`

and then install the specific modules as `chromium@testing`

File: editor/Dockerfile [hunk 2]
```diff
@@ -2,15 +2,34 @@
...
 
-RUN apk add --no-cache jq bash build-base git python nginx curl
-RUN curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C /
+RUN apk -U --no-cache --allow-untrusted add \
```
Comment (marcbachmann) [inline editor/Dockerfile]: What do we use the `--allow-untrusted` for?

File: editor/Dockerfile (hunk 1 above)
Comment (vhf) [reply editor/Dockerfile]: Will do

File: editor/Dockerfile (hunk 2 above)
Comment (vhf) [reply editor/Dockerfile]: Removed.

**Pull request merged**